packages/apt/test.sh

```diff
@@ -33,6 +33,16 @@ architecture=$(dpkg --print-architecture)
 wget \
   https://packages.groonga.org/${distribution}/groonga-apt-source-latest-${code_name}.deb
 sudo apt install -V -y ./groonga-apt-source-latest-${code_name}.deb
+
+case ${code_name} in
+  bookworm)
```

Comment:
We'll require Apache Arrow repository for all newer versions.

```suggestion
case ${distribution}-${code_name} in
  debian-bullseye|ubuntu-*)
    :
    ;;
  *)
```